contracts/Overtime/SportsAMMV2.sol

```diff
@@ -165,11 +172,21 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
             ISportsAMMV2RiskManager.RiskStatus riskStatus
         )
     {
-        (totalQuote, payout, fees, amountsToBuy, riskStatus) = _tradeQuote(_tradeData, _buyInAmount, true);
+        if (_collateral != address(0)) {
+            // getMinimumReceived returns the USD amount for the buyInAmount in the collateral (USD in 18 decimals)
+            // Note the method to obtain the collateralQuote can be modified to use (priceFeed.rateForCurrency * _buyInAmount)
+            // For using the price feed, it is better to merge this branch and add addressManager accross all contracts
+            collateralQuote = multiCollateralOnOffRamp.getMinimumReceived(_collateral, _buyInAmount);
+            // If USDC is default collateral needs to be transformed
+            // Note this better to be done on multiCollateralOnOffRamp side
+            collateralQuote = collateralQuote / (10 ** (18 - defaultCollateralDecimals));
```

Comment:
Not sure this is correct, if USDC is the default network collateral than I believe multicollateral contract also handles that

Reply:
It should be checked on live multicollateral contract on Arbitrum

Reply:
on live multicollateral returns 6
priceFeed returns 18

Reply:
this is pending, needs several changes in Mocks

Reply:
done
It will require extension of the PriceFeed contract